docs/rules/require-jsdoc.md

```diff
@@ -0,0 +1,56 @@
+# Require JSDoc comment (require-jsdoc)
+
+This rule generates warnings for nodes which do not have JSDoc comments. It considered a good practice to document the behavior of different nodes to help engineers understand the functionality of the node.
+Supported nodes:
+
+* `FunctionExpression`
```

Comment:
Hmm, not sure about `FunctionExpression` as a general part of this rule. Maybe try running this rule on the ESLint repo and see what sort of problems are reported? I think we might want to limit to methods, but running on some real code should help us figure that out.